doc/dev/developer_guide/index.rst

```diff
@@ -1586,6 +1586,148 @@ server list`` on the teuthology machine, but the target VM hostnames (e.g.
+                    filter should be the test you want to run
+
+.. note:: The priority number present in the command above is just a
+   placeholder. It might be highly inappropriate for the jobs you may want to
+   trigger. See `Testing Priority`_ section to pick a priority number.
+
+.. note:: Don't skip passing a priority number, the default value is 1000
+   which way too high; your job probably might never run.
+
+#. Wait for the tests to run. ``teuthology-suite`` prints a link to the
+   `Pulpito`_ page created for your tests.
+
+Other frequently used/useful options are ``-d`` (or ``--distro``),
+``--distroversion``, ``--filter-out``, ``--timeout``, ``flavor``, ``-rerun``.
+Run ``teuthology-suite --help`` to read description of these and every other
+options available.
+
+About Suites and Filters
+------------------------
+See `Suites Inventory`_ for a list of suites of integration tests present
+right now. Alternatively, each directory under ``qa/suites`` in Ceph
+repository is a integration test suite, so you can look within that directory
+to decide an appropriate argument for ``-s``.
+
+For picking an argument for ``--filter``, look within
+``qa/suites/<suite-name>/<subsuite-name>/tasks`` to get keywords for filtering
+tests. Each YAML file in there can trigger a bunch of tests; using the name of
+the file, without the extension part of the file name, as an argument to the
+``--filter`` will trigger those tests. For example, the sample command above
+uses ``cephfs-shell`` since there's a file named ``cephfs-shell.yaml`` in
+``qa/suites/fs/basic_functional/tasks/``. In case, the file name doesn't hint
+what bunch of tests would it trigger, you can look at the contents of the file
+for ``modules`` attribute. For ``cephfs-shell.yaml`` the ``mpdules`` attribute
+is ``tasks.cephfs.test_cephfs_shell`` which means it'll trigger all tests in
+``qa/tasks/cephfs/test_cephfs_shell.py``.
+
+Killing Tests
+-------------
+Sometimes your job might not complete running for several minutes or even
+hours even after tests you had trigged have completed running and other times
+you just might trigger few tests by mistake. You can terminate a job case as
+follows::
+
+        teuthology-kill -r teuthology-2019-12-10_05:00:03-smoke-master-testing-basic-smithi
+
+Let's call the the argument passed to ``-r`` as test ID. You can find it
+easily in the link to the Pulpito page for the test. For example, for the
+above test ID, the link is - http://pulpito.front.sepia.ceph.com/teuthology-2019-12-10_05:00:03-smoke-master-testing-basic-smithi/
+
+Re-running Tests
+----------------
+Pass ``--rerun`` option, with test ID as an argument to it, to
+``teuthology-suite`` command::
+
+    teuthology-suite -v -m smithi -c wip-rishabh-fs-test_cephfs_shell-fix -p 50 --rerun teuthology-2019-12-10_05:00:03-smoke-master-testing-basic-smithi
+
+The meaning of rest of the options is already covered before.
```

Comment:
This line can be dropped.

Reply:
It's nice to have, IMO, when someone directly jumps to this sub-section or when someone is skimming through docs.

Reply:
Since it is a `rerun` and options are defined few sub-sections up in the same page. This line can be skipped.